controllers/index.js

```diff
@@ -0,0 +1,24 @@
+const router = require('express').Router();
+let Exercise = require('../models/activity');
+
+
+
+router.route('/add').post((req, res) => {
+  const username = req.body.username;
```

Comment:
We cannot accept just username. The backend will need to receive the stored token from local storage. If this isn't set, anybody can pretend to be anybody.

Reply:
Well, I made those files before the register/login models and routes were implemented, just to get us started because I started the activity section in the front end already, but now if you go to the register branch, I modified the activity-log data to be accepted once the user token is provided, tried it on postman and it worked as it should, I still want to separate the routes for the users and the activity-log though but I keep getting an error when I require them in server.js as two separate files, so for now all routes are together in one file, it works but it's too cluttered. Please take a look at the register branch and guess it's ready to merge now, and will be thankful if you could correct any bugs you spot.